go.mod

```diff
@@ -2,9 +2,10 @@ module spotify
 
 go 1.16
 
+
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/brianstrauch/spotify v0.3.1
+	github.com/brianstrauch/spotify v0.3.2
```

Comment:
Yep, let's call this version v0.4.0 since there are new features.
https://semver.org

Reply:
Here you go: https://github.com/brianstrauch/spotify/releases/tag/v0.4.0